src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,71 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import cn from 'classnames';
+
+import { MovieCard } from '../MovieCard/MovieCard';
+import { Movie } from '../../types/Movie';
+import { getMovie } from '../../api';
+
+type Props = {
+  movies: Movie[];
+  onAdd: (newMovie: Movie) => void;
+};
+
+const defaultImg = 'https://via.placeholder.com/360x270.png?text=no%20preview';
+
+export const FindMovie: React.FC<Props> = ({ movies, onAdd }) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isErrorOnLoading, setErrorOnLoading] = useState(false);
+
+  const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    setIsLoading(true);
+
+    const loadedMovieData = await getMovie(query);
+
+    if ('Error' in loadedMovieData) {
+      setErrorOnLoading(true);
+    } else {
+      const newMovie = {
+        title: loadedMovieData.Title,
+        description: loadedMovieData.Plot,
+        imgUrl: loadedMovieData.Poster !== 'N/A'
+          ? loadedMovieData.Poster
+          : defaultImg,
+        imdbUrl: `https://www.imdb.com/title/${loadedMovieData.imdbID}`,
+        imdbId: loadedMovieData.imdbID,
+      };
+
+      setMovie(newMovie);
+    }
+
+    setIsLoading(false);
+  };
+
+  const clearFormOnAddMovie = () => {
+    setQuery('');
+    setMovie(null);
+  };
+
+  const moviesImdbId = movies.map(film => film.imdbId);
+
+  const handleAddMovie = (movieToAdd: Movie) => {
+    if (moviesImdbId.includes(movieToAdd.imdbId)) {
+      clearFormOnAddMovie();
+    } else {
+      onAdd(movieToAdd);
+      clearFormOnAddMovie();
+    }
```

Comment:
```suggestion
    if (!moviesImdbId.includes(movieToAdd.imdbId)) {
        onAdd(movieToAdd);
    } 
     clearFormOnAddMovie();
    }
```

Reply:
It would be cool if you show some message/error if I have this film on the list.